src/components/Onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { Image, SafeAreaView, View } from 'react-native';
+import Onboarding from 'react-native-onboarding-swiper';
+import { useNavigation } from '@react-navigation/native';
+import { useStyles } from 'react-native-unistyles';
+import { useTranslation } from 'react-i18next';
+
+import { styleSheet } from './Onboarding.style';
+import useGetOnboardingStatus from '@src/hooks/useGetOnboardingStatus';
+import { DotIndicator } from './OnboardingButtons/Indicator';
+import { Next } from './OnboardingButtons/Next';
+
+import Onboarding1 from '@assets/images/onboarding/onboarding1.png';
+import Onboarding2 from '@assets/images/onboarding/onboarding2.png';
+import Onboarding3 from '@assets/images/onboarding/onboarding3.png';
+
+export const OnboardingScreen = () => {
+	const { isFirstLaunch, isLoading: onboardingIsLoading } = useGetOnboardingStatus();
+	const { t } = useTranslation();
+	const { styles } = useStyles(styleSheet);
+	const navigation = useNavigation();
+
+	useEffect(() => {
+		if (!onboardingIsLoading && !isFirstLaunch) {
+			(navigation as any).navigate('login/index');
+		}
+	}, [isFirstLaunch, onboardingIsLoading, navigation]);
+
+	const setOnboardingStatus = async () => {
+		try {
+			await AsyncStorage.setItem('hasOnboardingSeen', 'false');
+		} catch (error) {
+			// Error saving data
+			console.log(error);
+		}
+	};
+	const onDone = () => {
+		setOnboardingStatus();
+		(navigation as any).navigate('login/index');
```

Comment:
The fact of using any here should have been very concerning for you. Try to use `const router = useRouter` from `expo-router` here to avoid `any`. Moreover, consider using `replace` method instead of `push/navigate` here, since you would like to get this screen replaces entirely rather then stay in navigation state, meaning that that you can "go back" here from other screens of your app